### DISCUSSION TITLE
Caddy configuration not found

### DISCUSSION THREAD

**Discussion by TuguldurJ** [postalserver/postal#2238] [Help with using Postal] [answered]
#2231 related issue. sometimes postal web server is down due to API requests. It happens when ~130 messages are sent per minute. 
Thus, I need to know how many API requests the postal server can handle? I would then configure the thread configuration on my own API request processing server.
Before, I modified mariadb connection pool and other database tuning configuration have been tested.  Nothing has changed in performance.
Furthermore, I could not find the postal web server configuration from docker containers.
I tried to locate the caddy source files or directory of the postal-caddy container which is `docker exec -u=0 -it postal-caddy /bin/bash`
Where are the web server configuration files? And how to tune the performance of incoming web server API requests? 

**Server:**
```
128gb ram, xeon 24core, 1gb/s public network bandwith, bare metal dedicated server
```
**Docker.yml:** 
```
worker: 
scale: 64
```
**Mariadb:**
```
innodb_buffer_pool_size = 80GB
innodb_buffer_pool_instances = 40
max_connections: = 100000
connect_timeout         = 31536000
wait_timeout            = 120000
interactive_timeout     = 120000
max_allowed_packet      = 256M
net_read_timeout        = 31536000
net_write_timeout       = 31536000
wait_timeout            = 31536000
innodb_flush_log_at_trx_commit = 2
thread_cache_size               = 8
```

**Comment by willpower232** [comment, marked as answer]
Without knowing the full actual response you're receiving it is difficult to know what the problem is.

The workers are for outgoing email, the web one will be receiving your API requests but I don't think you can scale that one without fighting for ports.

If you aren't getting much information then I guess you need to throttle at less than 130 messages. There are no artificial tuning options available that I am aware of, you only need to change your database settings to deal with larger individual messages as far as I can remember.
